std_msgs/Header header 
float64[] range
float64[] bearing
uint32 num_neighbors